tests/integrate-helios-lucid.test.ts

```diff
@@ -0,0 +1,137 @@
+import { describe, expect, it, vi } from 'vitest'
+import { promises as fs } from 'fs';
+import {
+  Assets, 
+  ConstrData, 
+  MintingPolicyHash,
+  NetworkEmulator,
+  NetworkParams,
+  Program, 
+  Value, 
+  textToBytes,
+  TxOutput,
+  Tx, 
+} from "@hyperionbt/helios";
+
+import {
+  Assets,
+  Data,
+  Emulator,
+  fromText,
+  generateSeedPhrase,
+  getAddressDetails,
+  Lucid,
+  SpendingValidator,
+  toUnit,
+  TxHash,
+} from "lucid-cardano"; // NPM
+
+describe('introduces Helios, a source of a new template', () => {
+
+	const main = async () => {
+
+	try {
+	// https://github.com/spacebudz/lucid/blob/main/tests/emulator.test.ts
+	async function generateAccount(assets: Assets) {
+	  const seedPhrase = generateSeedPhrase();
+	  return {
+	    seedPhrase,
+	    address: await (await Lucid.new(undefined, "Custom"))
+	      .selectWalletFromSeed(seedPhrase).wallet.address(),
+	    assets,
+	  };
+	}
+
+	const lender = await generateAccount({ lovelace: 75000000000n });
+	const borrower = await generateAccount({ lovelace: 100000000n });
+
+	const emulator = new Emulator([lender, borrower]);
+
+	const lucid = await Lucid.new(emulator);
+
+	// emulator state changes, how do I encapsulate this?
+	lucid.selectWalletFromSeed(borrower.seedPhrase);
+	const zeroState = await lucid.wallet.getUtxos(lucid.wallet.address())
+	// https://github.com/spacebudz/lucid/blob/main/src/examples/matching_numbers.ts
+	
+	const src = await fs.readFile('./src/always-succeeds.hl', 'utf8');
```

Comment:
now I have a plutus script

Reply:
changing script here implies different structure of code, there is not much reusability